sdk/ml/azure-ai-ml/azure/ai/ml/entities/_inputs_outputs/output.py

```diff
@@ -23,7 +23,7 @@ class Output(_InputOutputBase):
     @overload
     def __init__(
         self,
-        type: Any,
+        type: Literal["uri_folder"] = "uri_folder",
```

Comment:
Same as above, this overload appears to be a duplicate of the one below, so can probably be removed.